gradle/libs.versions.toml

```diff
@@ -6,6 +6,7 @@ allure = "2.4.0"
 compose = "1.0.5"
 activityCompose = "1.4.0"
 androidXTest = "1.4.0"
+testOrchestrator = "1.4.1"
```

Comment:
Do we need testOrchestrator?
I've (and not only me) observed a lot of cases when Orchestrator "doesn't see" tests while you try to run some bunch of them. Temporary removing of Orchestrator from the dependencies and usage of base AndroidTestRun resolved the issue for a lot of cases. Let's try to remove Orchestrator because anyway we use Marathon or something similar for more smart tests execution.

Reply:
Were there any problems with the app state being shared between tests inside the test run?

Reply:
@matzuk I've encountered this behavior too, but it got resolved by upgrading orchestrator from 1.4.0 to 1.4.1. 
@eakurnikov Yes, without orchestrator some tests worked fine when I launched them in scope of their module (gradlew moduleName:connectedDebugAndoirdTest) but failed when I run them in all modules (gradlew connectedAndroidDebugTest)

Reply:
1.4.0 doesn't work with 31 api at all. Even after the update orchestrator remains flaky especially when it comes to local test run. But I think it is fine to have it here for now. We can add other runner in future.